python/setup.py.in

```diff
@@ -23,6 +23,21 @@ with open('@PADDLE_SOURCE_DIR@/python/requirements.txt') as f:
 if '${CMAKE_SYSTEM_PROCESSOR}' not in ['arm', 'armv7-a', 'aarch64']:
     setup_requires+=["opencv-python"]
 
+# the prefix is sys.prefix which should always be usr
+paddle_bin_dir = 'local/opt/paddle/bin'
```

Comment:
How about not in docker? Still like `local/local` ?
Or could you set --prefix=/usr  when setup docker.

Because I thought outside docker, it should be default as /usr. I tried on my machine and just found that in @luotao1 's machine the path is still look right to me:

> /home/luotao/.jumbo/local/opt/paddle/bin/paddle_merge_model
  /home/luotao/.jumbo/local/opt/paddle/bin/paddle_pserver_main
  /home/luotao/.jumbo/local/opt/paddle/bin/paddle_trainer
  /home/luotao/.jumbo/local/opt/paddle/bin/paddle_usage

Her case is under `/home/luotao/.jumbo`.